Kitodo/src/main/java/org/kitodo/production/model/LazyHitModel.java

```diff
@@ -54,14 +55,28 @@ public int getRowCount() {
     public List<Object> load(int first, int resultSize, String sortField, SortOrder sortOrder, Map filters) {
 
         searchResult = ServiceManager.getImportService().performSearch(
-                this.selectedField, this.searchTerm, this.selectedCatalog, first, resultSize);
+                this.selectedField, getSearchTermWithDelimiter(this.searchTerm), this.selectedCatalog, first, resultSize);
 
         if (Objects.isNull(searchResult) || Objects.isNull(searchResult.getHits())) {
             return Collections.emptyList();
         }
         return searchResult.getHits().stream().map(r -> (Object)r).collect(Collectors.toList());
     }
 
+    /**
+     * Returns the searchTerm with configured Delimiter.
+     * @param searchTerm the searchterm to add delimiters.
+     * @return searchTermWithDelimiter
+     */
+    public String getSearchTermWithDelimiter(String searchTerm) {
```

Comment:
Could this method be moved to `ImportService` and be called inside `performSearch` there instead? `ImportService` already imports `OPACConfig` and I think it would be cleaner if the `LazyHitModel` itself wouldn`t need to manipulate any of the search parameters passed by the user but instead would let the `ImportService` handle all that (which is already the case with all other catalog specific settings).